src/features/session_trace/aggregate/index.js

```diff
@@ -237,43 +233,43 @@ export class Aggregate extends FeatureBase {
     return origin
   }
 
+  // Tracks when the window history API specified by wrap-history is used.
   storeHist (path, old, time) {
-    var node = {
+    const node = {
       n: 'history.pushState',
       s: time,
       e: time,
       o: path,
       t: old
     }
-
     this.storeSTN(node)
   }
 
+  // Processes all the PerformanceResourceTiming entries captured (by observer).
   storeResources (resources) {
     if (!resources || resources.length === 0) return
 
     resources.forEach((currentResource) => {
-      var parsed = parseUrl(currentResource.name)
-      var res = {
+      if (currentResource.fetchStart <= this.laststart) return // don't recollect already-seen resources
```

Comment:
this was comparing `.fetchStart | 0` before, is this operation no longer necessary?

Reply:
Oh thanks for noticing that. I'm not sure when the old `PT.fetchStart` would ever be undefined, but I'll just add it back here anyways.